### PR TITLE
ci: update test matrix

### PR DESCRIPTION
**Pull request by kortschak**
Please take a look.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ branches:
 
 environment:
   matrix:
-    - GOROOT: 'c:\go19'
     - GOROOT: 'c:\go110'
     - GOROOT: 'c:\go111'
+    - GOROOT: 'c:\go112'
```
Comment (sbinet) [inline appveyor.yml]: reverse order as for travis?

Comment (kortschak) [reply appveyor.yml]: Another PR.

**Comment by kortschak** [comment]
Can you restart the AppVeyor tests?

**Comment by kortschak** [comment]
Thanks

**Pull request merged**